README.md

```diff
@@ -0,0 +1,99 @@
+# Kinetic Convolution of Polygonal Tracings
+This repository offer an implementation of the concept of kinetic
+convolution of polygonal tracings, as introduced in [Gui83] and
+extended in [Mil07].
+
+![kinetic convolution of two polygonal tracings](/images/minkowski-sum.png)
+
+# Usage
+Add the NuGet reference to the project:
+```PowerShell
+    dotnet add package Hilke.KineticConvolution --version 0.1.0
+```
+
+and import the namespace in your source file:
+```C#
+    using Hilke.KineticConvolution;
+```
+
+If you want to use the implementation of algebraic number over the
```

Comment:
"instead of `double` data type" sounds better to me (but I am not a native English speaker)

Reply:
C'est pas exectement ca. C'est vrai que ma formulation initiale etait pas super, mais je propose encore autre chose, pour finir. Je te laisse voir si ca joue pour toi.

Reply:
Pas de nouvelles, je ferme.